### ISSUE TITLE
Machine Learning Algorithms simple Tutorials

### ISSUE THREAD

**Issue by Moddy2024** [meks-0202/Python-Tools#13]
More Machine Learning Algorithms simple examples are needed

* [] Random Forest Classifier
* [] Random Forest Regressor
* [x] XGBoost Classifier
* [] XGBoost Regressor
* [] Decision Tree
* [] Cat Boost Classifier
* [] Cat Boost Regressor
* [] Logistic Regression
* [] Linear Regression
* [] SVM Classifier
* [] SVM Regressor
* [] KNN
* [] Neural Network

Add more to the list.

The tick ones are already contributed.

**Comment by Moddy2024** [comment]
@meks-0202  Can you please add the tag of hacktoberfest so it easily pops on searches when people are searching for issues. They can contribute to it too.

**Comment by meks-0202** [comment]
@Moddy2024 I have already added the labels

**Comment by Moddy2024** [comment]
> @Moddy2024 I have already added the labels

Yeah you added at the repo. I am saying that to add at the issue because some people search issue with keywords and it will pop up at more people searches and actually this issue isn't solved yet there are more algorithms in the list.
